### DISCUSSION TITLE
Speedybee F405 v3 INAV support?

### DISCUSSION THREAD

**Discussion by girgine** [iNavFlight/inav-configurator#1703] [Q&A]
Hi all, I can flash my FCC with 5.x without any issues (holding the button to enabe DFU mode first). After the flashing though, I can no longer access to FCC through the USB port 9getting time out error). 

INAV has also a firmware for F405 and I can flash it and USB works but its not compatible with the V3 so I cant use it. 

I am using the latest firmware for this FCC through the configurator. 

any suggestions?

thanks

**Comment by Kgray44** [comment]
Hello @girgine 

Funny enough, I just got a SpeedyBee F405 V3 with ESC stack and am using it right now!  I am using configurator version 6.0.0, and fc firmware version 6.0.0, and everything seems to be working just fine (with some minor issues though https://github.com/iNavFlight/inav-configurator/issues/1749).

What button are you holding?  You shouldn't be holding the BOOT button down.

What configurator version are you using?

Try flashing fc firmware version 6.0.0 _without_ holding any buttons down.

Have you tried any of these troubleshooting steps?
<img width="1053" alt="No Response Troubleshooting" src="https://user-images.githubusercontent.com/64373011/228565197-a522a894-e8b2-472b-bede-e24f82389753.png">

Hope this helps!
